guides/source/configuring.md

```diff
@@ -535,7 +535,7 @@ There are a few configuration options available in Active Support:
 
 * `config.active_support.time_precision` sets the precision of JSON encoded time values. Defaults to `3`.
 
-* `config.active_support.halt_callback_chains_on_return_false` specifies whether ActiveRecord, ActiveModel and ActiveModel::Validations callback chains can be halted by returning `false` in a 'before' callback. Defaults to `true`.
+* `ActiveSupport.halt_callback_chains_on_return_false` specifies whether Active Record, Active Model and Active Model validations callback chains can be halted by returning `false` in a 'before' callback. Defaults to `true`.
```

Comment:
Maybe it should just be `Active Record and Active Model` instead of `Active Record, Active Model and Active Model validations`. Same for other occurrences in this PR.